### PR TITLE
Battle Arena now allows users to correctly challenge each other for 3v3s

### LINKED ISSUE #411: Implement front end handling for 3v3 battles.
**Describe the solution you'd like**
Have toggles for 1v1 and 3v3 in Training Arena and Battle Arena. Needs to render the three tanks for the user. The size of the rendering can be changed with the TankDisplay.css.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by blaiadam**
**Description**
Battle Arena now correctly differentiates between the 3v3 and 1v1 challenges and users can now fully challenge each other in 3v3 battles.

**Resolves These Issues**
Resolves #411  finally

**Type of change**
Check options that are relevant.

- [ ] Bug fix (fixes a bug issue)
- [x] New feature (adds functionality)
- [ ] This fix or feature will cause existing functionality to not work as expected. (Please elaborate in Description.)

**Steps to Verify Functionality**
1. Make a 3v3 wager.
2. Challenge on another account.

**Final Checklist:**
Go down the list and check them off.

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my own code
- [x] My changes pass Flow, build without errors and (if applicable) pass Jest tests.
- [ ] This change requires a update in the design document (if applicable)

### CODE REVIEW

File: src/backend/routes/battleRoutes.js [hunk 1]
```diff
@@ -30,9 +30,8 @@ router.post('/prepareMatch1v1', [
 // Body: challengerTankIds and personBeingChallengedId
 // Returns the newly created batlleRecordId
 router.post('/prepareMatch3v3', [
-	check('challengerTankIds', 'Need 3 tank Ids')
-		.isArray({ min: 3, max: 3})
-		.isMongoId(),
+	check('challengerTankIds')
```
Comment (SecondThread) [inline src/backend/routes/battleRoutes.js:33]: Are we sure that this change is a good idea?

**Review by SecondThread — APPROVED** [review]
Looks good for the most part, just one question about whether removing some of the backend checks was a good idea. If there was a reason for it, I am fine with it, but I just want to make sure it was intended.

File: src/backend/routes/battleRoutes.js (hunk 1 above)
Comment (Derplime) [reply src/backend/routes/battleRoutes.js:33]: Why was the array check removed?

Comment (Derplime) [reply src/backend/routes/battleRoutes.js:33]: oh nvm it's there. the MongoId check is removed or it will conflict with turning in arrays with `null`

Comment (Derplime) [reply src/backend/routes/battleRoutes.js:33]: Though it would be nice to keep some message instead of removing it

**Pull request merged**